release/nightly_tests/nightly_tests.yaml

```diff
@@ -0,0 +1,16 @@
+- name: shuffle_10gb
+  owner:
+    mail: "ekl@anyscale.com"
+    slack: "@proj-data-processing"
+
+  cluster:
+    app_config: shuffle/shuffle_app_config.yaml
+    compute_template: shuffle/shuffle_compute.yaml
+
+  run:
+    timeout: 600
+    script: python shuffle/shuffle_10gb.py
```

Comment:
@krfricke would it also work if I changed this to `python -m ray.experimental.shuffle --ray-address=auto --num-partitions=50 --partition-size=200e6`? That would eliminate the need for an intermediate script.

Reply:
Running the script should work, but for metrics other than pass/fail the module would have to write a json still

Reply:
Ok, I might do that in a followup PR